beaglebone-pocket.coffee

```diff
@@ -0,0 +1,44 @@
+deviceTypesCommon = require '@resin.io/device-types/common'
+{ networkOptions, commonImg, instructions } = deviceTypesCommon
+
+BBB_BOOT = 'Insert SD CARD into <%= TYPE_NAME %>. Power up the <%= TYPE_NAME %>.
+You should now see the blue LEDs start flashing.'
+
+module.exports =
+	version: 1
+	slug: 'beaglebone-pocket'
+	name: 'PocketBeagle'
+	arch: 'armv7hf'
+	state: 'experimental'
+
+	instructions: [
```

Comment:
If you remove the above `BBB_BOOT` then this `instructions` section can be replaced simply by:
> instructions: commonImg.instructions